### PR TITLE
Replaced all the css into tailwind css on TimelineEvent component

### PR DESCRIPTION
**Pull request by subhajit20**
# Description of changes

Replaced all the css into tailwind css on TimelineEvent component
Issue - #1725 

# Issue Resolved

Fixes #NA

### CODE REVIEW

File: components/Timeline/TimelineEvent/TimelineEvent.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ TimelineEvent.propTypes = {
 
 function TimelineEvent({ content, title }) {
   return (
-    <div className={styles.eventContainer}>
-      <h4 className={styles.eventTitle}>{title}</h4>
+    <div className="ml-8 mr-0 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
+      <h4 className="uppercase md:text-2xl">{title}</h4>
 
       <div className={styles.eventContent}>{content}</div>
```
Comment (subhajit20) [inline components/Timeline/TimelineEvent/TimelineEvent.js:14]: what about this className? I did not find it in the css file. @kylemh

Comment (kylemh) [reply components/Timeline/TimelineEvent/TimelineEvent.js:14]: huh - i guess there are no styles then 🤷🏼

File: components/Timeline/TimelineEvent/TimelineEvent.js [hunk 2]
```diff
@@ -8,8 +8,8 @@ TimelineEvent.propTypes = {
 
 function TimelineEvent({ content, title }) {
   return (
-    <div className={styles.eventContainer}>
-      <h4 className={styles.eventTitle}>{title}</h4>
+    <div className="ml-8 mr-0 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
+      <h4 className="uppercase md:text-2xl">{title}</h4>
```
Comment (kylemh) [inline components/Timeline/TimelineEvent/TimelineEvent.js:12]:
```suggestion
      <h4 className="capitalize md:text-2xl">{title}</h4>
```

File: components/Timeline/TimelineEvent/TimelineEvent.js [hunk 3]
```diff
@@ -8,8 +8,8 @@ TimelineEvent.propTypes = {
 
 function TimelineEvent({ content, title }) {
   return (
-    <div className={styles.eventContainer}>
-      <h4 className={styles.eventTitle}>{title}</h4>
+    <div className="ml-8 mr-0 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
```
Comment (kylemh) [inline components/Timeline/TimelineEvent/TimelineEvent.js:11]:
```suggestion
    <div className="mb-8 border-1 border-solid border-[#c5c5c5] p-5 rounded-md">
```

in a 4-digit value, it reads like a clock (top, right, bottom, left)

File: components/Timeline/TimelineEvent/TimelineEvent.js (hunk 2 above)
Comment (subhajit20) [reply components/Timeline/TimelineEvent/TimelineEvent.js]: done

File: components/Timeline/TimelineEvent/TimelineEvent.js (hunk 3 above)
Comment (subhajit20) [reply components/Timeline/TimelineEvent/TimelineEvent.js]: done

**Review by kylemh — APPROVED** [review]

**Pull request merged**